assets/js/theme/global/cart-preview.js

```diff
@@ -69,7 +69,12 @@ export default function (secureBaseUrl, cartId) {
         const cartQtyPromise = new Promise((resolve, reject) => {
             utils.api.cart.getCartQuantity({ baseUrl: secureBaseUrl, cartId }, (err, qty) => {
                 if (err) {
-                    reject(err);
+                    // If this appears to be a 404 for the cart ID, set cart quantity to 0
+                    if (err === 'Not Found') {
```

Comment:
Why the messy string comparison?

Our request implementation only returns the string of the response code, not the actual error object:

https://github.com/bigcommerce/stencil-utils/blob/master/src/lib/request.js#L127

As such, I have nothing else to match on without a significant refactor.

Fortunately, the fallback behavior here is pretty elegant.